src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -165,7 +167,13 @@ const InnerForm: React.FC<FormikProps<FormValues>> = props => {
         )}
       </Flex>
 
-      <Button mt={1} size="large" width="100%" loading={isSubmitting}>
+      <Button
+        mt={1}
+        size="large"
+        width="100%"
+        loading={isSubmitting}
+        {...{ type: "submit" } as any}
```

Comment:
We need `type="submit"` according to a console warning but it is not available on a palette button 🙃

Reply:
PRs welcome 😉

Reply:
My understanding has been that we don't put html-specific attrs on palette components- what is the right way to do this?

Reply:
That's not a strict constraint. We just don't want to add HTML props that flow down to react native.

Reply:
Let's not worry with this right now. It's fine for this.